### PR TITLE
Change order of class

### PR DESCRIPTION
**Pull request by krazyeom**
Other class that is added newly is ordered lastly on http://www.getmdl.io/components/index.html#buttons-section

but only this class is ordered in the middle of class tag.

It should be ordered lastly.

Before:

```
<!-- Colored FAB button -->
<button class="mdl-button mdl-js-button mdl-button--fab mdl-button--colored">
  <i class="material-icons">add</i>
</button>
<!-- Colored FAB button with ripple -->
<button class="mdl-button mdl-js-button mdl-button--fab mdl-js-ripple-effect mdl-button--colored">
  <i class="material-icons">add</i>
</button>
```

After:

```
<!-- Colored FAB button -->
<button class="mdl-button mdl-js-button mdl-button--fab mdl-button--colored">
  <i class="material-icons">add</i>
</button>
<!-- Colored FAB button with ripple -->
<button class="mdl-button mdl-js-button mdl-button--fab mdl-button--colored mdl-js-ripple-effect">
  <i class="material-icons">add</i>
</button>
```

### CODE REVIEW

File: src/button/snippets/raised-ripple-accent.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Accent-colored raised button with ripple -->
-<button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">
+<!-- Accent-colored raised button -->
+<button class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent">
```
Comment (surma) [inline src/button/snippets/raised-ripple-accent.html]: Why did you remove the ripple class here?

Comment (addyosmani) [inline src/button/snippets/raised-ripple-accent.html]: The ripple class should be staying here. Are we missing something @krazyeom?

**Comment by Garbee** [comment]
What does the order have to do with anything in the first place? I don't see what problem this is fixing.

**Comment by surma** [comment]
Consistency and didactics, mostly

**Comment by Garbee** [comment]
There is also `mdl-js-button` as the second class. So should all JS classes be moved to the end of examples? Consistent within itself perhaps, but not as a whole.

File: src/button/snippets/raised-ripple-accent.html (hunk 1 above)
Comment (krazyeom) [inline src/button/snippets/raised-ripple-accent.html]: I am so sorry about that commit number 57cf893 and  335449b

**Comment by krazyeom** [comment]
I am not sure which is best order of class  for the example. maybe I am wrong.

**Comment by Garbee** [comment]
I mean, if we go for consistency where js classes are at the end, then this helps to start aligning that. I just don't see a real need since classes defined in HTML have no bearing by order. In CSS is when order matters (last defined wins) then specificity of selector.

**Comment by krazyeom** [comment]
@Garbee I agree with you. 
This is a starter guide. So I think it is necessary only for the guide.
